data/json/mutations/mutations.json

```diff
@@ -5881,7 +5884,7 @@
     "visibility": 7,
     "ugliness": 4,
     "mixed_effect": true,
-    "description": "Your arms have transformed into tentacles, resulting in a bonus of 1 to Dexterity, permanent hand encumbrance of 30, and inability to wear gloves.  Somewhat decreases wet penalties.",
+    "description": "Your arms have transformed into tentacles, resulting in +1 bonus to Dexterity, permanent hand encumbrance of 30, and inability to wear gloves.  Somewhat decreases wet penalties.",
```

Comment:
```suggestion
    "description": "Your arms have transformed into tentacles, resulting in a +1 to Dexterity, permanent hand encumbrance of 30, and an inability to wear gloves.  Somewhat decreases wet penalties.",
```

Reply:
The word "bonus" is used throughout the file. I think it'd be better kept, to maintain consistency, no?

Reply:
OK. ("a +1 bonus to Dexterity", then?)

Reply:
Huh. I managed to miss that one, and only that one.

Reply:
Happens to everyone!

Re "bonus": And, if while using StatsThroughSkills or similar, what's required for stat gain goes up with the existing stat level, and this doesn't happen (or isn't supposed to happen!) with mutations like this one, then it makes sense to call it a "bonus". In cases where this is not true, perhaps "increases [stat] by 1"?

Reply:
I wouldn't know how to tell when the case was true or not.

Reply:
I can make a guess, but C++ inheritance rules sometimes mess me up in figuring out things like that.